app/src/main/res/layout/activity_text_enter.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.secuso.privacyFriendlyCodeScanner.qrscanner.QRGenerating.TextEnterActivity">
+
+
+    <Button
+        android:id="@+id/generate"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:layout_margin="3dp"
+        android:layout_marginBottom="42dp"
+        android:layout_toRightOf="@+id/imageView"
+        android:layout_weight="1"
+        android:background="@drawable/button_fullwidth"
+        android:padding="1dp"
+        android:text="Generate"
+        android:textColor="@android:color/white" />
+
+    <EditText
+        android:id="@+id/editText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="160dp"
+        android:ems="10"
+        android:hint="Hello World"
+        android:inputType="textPersonName" />
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignStart="@+id/generate"
+        android:layout_alignBottom="@+id/editText"
+        android:layout_marginBottom="60dp"
+        android:text="The text you want to encode:" />
```

Comment:
I know the other files do not do this, but could you change this to use localized string resources instead of hardcoded ones?

Reply:
fixed with
- af63da42264e2ac5abd0380d124639c44efb4e32
and 
- c0ae6ab3aac780ca6a09f47bd173110985799033